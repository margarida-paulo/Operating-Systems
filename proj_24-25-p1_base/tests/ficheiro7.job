WRITE [(Gustavo,Lima)(Carlos,Vinícius)]
WRITE [(Nicolas,Mendes)(Alberto,Pereira)]
WRITE [(Fabiana,Alves)(Adriana,Lopes)]
WRITE [(Cristina,Sousa)(Alexandre,Rodrigues)]
WRITE [(Paulo,Carvalho)(Renata,Teixeira)]
WRITE [(Helena,Monteiro)(Marco,Ferreira)]
WRITE [(Fernando,Gomes)(Patrícia,Barros)]
WRITE [(Rafael,Martins)(Camila,Ribeiro)]
WRITE [(Eduarda,Silva)(Sérgio,Oliveira)]
WRITE [(Amanda,Santos)(Hugo,Pires)]
WRITE [(Luciana,Nogueira)(Ricardo,Mendes)]
WRITE [(Jéssica,Costa)(Rogério,Lima)]
WRITE [(Rodrigo,Pinto)(Larissa,Amorim)]
WRITE [(Tatiana,Vieira)(Diego,Marques)]
WRITE [(Gisele,Ferreira)(Juliano,Rocha)]
WRITE [(Douglas,Ramos)(Viviane,Lima)]
WRITE [(Anderson,Moura)(Kelly,Neves)]
WRITE [(Fábio,Castro)(Cláudia,Barbosa)]
WRITE [(Mariana,Matos)(Márcio,Coelho)]
WRITE [(Júnior,Bastos)(Carolina,Gomes)]
SHOW
