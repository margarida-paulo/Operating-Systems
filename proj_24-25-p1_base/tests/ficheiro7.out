(Anderson,Moura)
(Amanda,Santos)
(Alexandre,Rodrigues)
(Adriana,Lopes)
(Alberto,Pereira)
(Amber,Heard)
(Anne,Hathaway)
(Angelina,Jolie)
(Amy,Adams)
(Albert,Camus)
(Aldous,Huxley)
(Agatha,Christie)
(Ben,Affleck)
(Brad,Pitt)
(Carolina,Gomes)
(Cláudia,Barbosa)
(Camila,Ribeiro)
(Cristina,Sousa)
(Carlos,Vinícius)
(Chadwick,Boseman)
(Carrie,Moss)
(Christian,Bale)
(Chris,Hemsworth)
(Chinua,Achebe)
(C.S.,Lewis)
(Charles,Dickens)
(Clarice,Lispector)
(Douglas,Ramos)
(Diego,Marques)
(Daniel,Radcliffe)
(Denzel,Washington)
(Eduarda,Silva)
(Emma,Watson)
(Ernest,Hemingway)
(Edgar,Allan)
(Emily,Dickinson)
(Fábio,Castro)
(Fabiana,Alves)
(Fyodor,Dostoevsky)
(Franz,Kafka)
(Fernando,Gomes)
(Gisele,Ferreira)
(Gustavo,Lima)
(Gal,Gadot)
(George,Orwell)
(Gabriel,Marquez)
(Hugo,Pires)
(Hugh,Jackman)
(Helena,Monteiro)
(Haruki,Murakami)
(Herman,Melville)
(Isabel,Allende)
(Italo,Calvino)
(Júnior,Bastos)
(Juliano,Rocha)
(Jéssica,Costa)
(Jason,Momoa)
(Joaquin,Phoenix)
(Johnny,Depp)
(Julia,Roberts)
(James,Joyce)
(J.R.R.,Tolkien)
(J.K.,Rowling)
(Jane,Austen)
(José,Saramago)
(Jorge,Luis)
(Kelly,Neves)
(Keanu,Reeves)
(Kate,Winslet)
(Larissa,Amorim)
(Luciana,Nogueira)
(Lupita,Nyong)
(Leonardo,DiCaprio)
(Liam,Nelson)
(Leo,Tolstoy)
(Luís,Camões)
(Márcio,Coelho)
(Marco,Ferreira)
(Matt,Damon)
(Morgan,Freeman)
(Margot,Robbie)
(Meryl,Streep)
(Margarida,Maggie)
(Marcel,Proust)
(Mark,Twain)
(Miguel,Cervantes)
(Mariana,Matos)
(Nicolas,Mendes)
(Nicole,Kidman)
(Natalie,Portman)
(Oscar,Wilde)
(Patrícia,Barros)
(Paulo,Carvalho)
(Pablo,Neruda)
(Rodrigo,Pinto)
(Rogério,Lima)
(Ricardo,Mendes)
(Rafael,Martins)
(Renata,Teixeira)
(Rachel,McAdams)
(Ryan,Gosling)
(Reese,Witherspoon)
(Robert,Downey)
(Sérgio,Oliveira)
(Sandra,Bullock)
(Scarlett,Johansson)
(Simone,Oliveira)
(Stephen,King)
(Sophia,Andresen)
(Tatiana,Vieira)
(Tom,Hanks)
(Toni,Morrison)
(Viviane,Lima)
(Victor,Hugo)
(Virginia,Woolf)
(Will,Smith)
(William,Faulkner)
(Zadie,Smith)
